e2e/test/scenarios/admin/databases/default-sample-database.cy.spec.js

```diff
@@ -0,0 +1,250 @@
+    cy.findByText("Hourly").click();
+    popover().within(() => {
+      cy.findByText("Daily").click({ force: true });
+    });
+
+    // "lets you change the cache_field_values period"
+    cy.findByLabelText("Never, I'll do this manually if I need to").should(
+      "have.attr",
+      "aria-selected",
+      "true",
+    );
+
+    cy.findByLabelText("Regularly, on a schedule")
+      .click()
+      .within(() => {
+        cy.findByText("Daily").click();
+      });
+    popover().findByText("Weekly").click();
+
+    cy.button("Save changes").click();
+    cy.wait("@databaseUpdate").then(({ response: { body } }) => {
+      expect(body.details["let-user-control-scheduling"]).to.equal(true);
+      expect(body.schedules.metadata_sync.schedule_type).to.equal("daily");
+      expect(body.schedules.cache_field_values.schedule_type).to.equal(
+        "weekly",
+      );
+    });
+    cy.button("Success");
+
+    // "lets you change the cache_field_values to 'Only when adding a new filter widget'"
+    cy.findByLabelText("Only when adding a new filter widget").click();
+    cy.button("Save changes", { timeout: 10000 }).click();
+    cy.wait("@databaseUpdate").then(({ response: { body } }) => {
+      expect(body.is_full_sync).to.equal(false);
+      expect(body.is_on_demand).to.equal(true);
+    });
+
+    // and back to never
+    cy.findByLabelText("Never, I'll do this manually if I need to").click();
+    cy.button("Save changes", { timeout: 10000 }).click();
+    cy.wait("@databaseUpdate").then(({ response: { body } }) => {
+      expect(body.is_full_sync).to.equal(false);
+      expect(body.is_on_demand).to.equal(false);
+    });
+  });
+
+  it("database actions sidebar", () => {
+    cy.intercept("POST", `/api/database/${SAMPLE_DB_ID}/sync_schema`).as(
+      "sync_schema",
+    );
+    cy.intercept("POST", `/api/database/${SAMPLE_DB_ID}/rescan_values`).as(
+      "rescan_values",
+    );
+    cy.intercept("POST", `/api/database/${SAMPLE_DB_ID}/discard_values`).as(
+      "discard_values",
+    );
+    cy.intercept("GET", `/api/database/${SAMPLE_DB_ID}/usage_info`).as(
+      `usage_info`,
+    );
+    cy.intercept("DELETE", `/api/database/${SAMPLE_DB_ID}`).as("delete");
+    // model
+    cy.request("PUT", "/api/card/1", { dataset: true });
+    // Create a segment through API
+    cy.request("POST", "/api/segment", {
+      name: "Small orders",
+      description: "All orders with a total under $100.",
+      table_id: ORDERS_ID,
+      definition: {
+        "source-table": ORDERS_ID,
+        aggregation: [["count"]],
+        filter: ["<", ["field", ORDERS.TOTAL, null], 100],
+      },
+    });
+    // metric
+    cy.request("POST", "/api/metric", {
+      name: "Revenue",
+      description: "Sum of orders subtotal",
+      table_id: ORDERS_ID,
+      definition: {
+        "source-table": ORDERS_ID,
+        aggregation: [["sum", ["field", ORDERS.SUBTOTAL, null]]],
+      },
+    });
```

Comment:
This is an improvement over the previous version of the "Should be able to delete database" tests.
It was originally making sure that related/affected questions will be deleted, but now it does that additionally for models, metrics and segments.